concepticondata/conceptlists/Goz-2017-600.tsv

```diff
@@ -0,0 +1,601 @@
+Goz-2017-600-470	470	somut	concrete			4.14	2.8	3.23	5.01	5.25	3.35	3.18	64	2	5
+Goz-2017-600-471	471	sondaj	drilling			5.32	3.55	6.26	5.32	5.36	3.69	5.77	5	2	6
+Goz-2017-600-472	472	sonsuz	endless			3.47	2.28	3.79	4.06	3.76	2.44	1.51	100	2	6
+Goz-2017-600-473	473	sos	sauce	132	SAUCE	3.58	2.04	3.11	4.35	4.61	5.37	6.64	100	1	3
+Goz-2017-600-474	474	soytarı	clown			3.03	2.37	3.34	3.63	2.84	5.56	5.24	1	3	7
+Goz-2017-600-475	475	soyut	abstract			4.4	2.93	3.6	5.55	5.31	2.48	1.51	55	2	5
+Goz-2017-600-476	476	sörf	surf			4.49	2.71	3.86	5.26	5.87	5.53	6.3	9	1	4
+Goz-2017-600-477	477	standart	standard			4.47	2.8	4.46	5.18	5.33	2.86	2.47	153	3	8
+Goz-2017-600-478	478	su	water	948	WATER	1.26	1.17	1.21	1.37	1.31	6.49	6.8	1537	1	2
+Goz-2017-600-479	479	suç	crime	1066	CRIME	2.4	1.96	2.71	2.8	2.19	3.45	2.48	221	1	3
+Goz-2017-600-480	480	suçlu	criminal			2.41	2.08	2.66	2.77	2.2	3.74	4.69	58	2	5
+Goz-2017-600-481	481	sulak	marshy			3.25	2.71	4.03	3.69	2.66	3.41	5.04	1	2	5
+Goz-2017-600-482	482	sülün	pheasant	3060	PHEASANT	4.8	3.56	5.51	5.14	4.28	2.45	5.53	4	2	5
+Goz-2017-600-483	483	süngü	bayonet			3.88	2.77	4.92	4.15	3.46	4.82	6.55	1	2	5
+Goz-2017-600-484	484	süresiz	infinite			3.58	2.42	3.42	4.11	4.28	1.72	2.15	3	3	7
+Goz-2017-600-485	485	sürpriz	surprise			2.38	1.65	1.79	2.83	3.09	3.43	2.71	49	2	7
+Goz-2017-600-486	486	süt	milk	635	MILK	1.36	1.28	1.31	1.44	1.39	6.42	6.79	200	1	3
+Goz-2017-600-487	487	şahane	marvelous			2.88	2.25	2.84	3.2	3.18	3.23	2.02	11	3	6
+Goz-2017-600-488	488	şair	poet	1045	POET	2.9	2.23	3.14	3.37	2.89	4.99	5.59	125	2	4
+Goz-2017-600-489	489	şalter	switch (electric)			4.29	2.77	4.45	4.67	4.86	5.09	6.73	6	2	6
+Goz-2017-600-490	490	şamata	commotion			3.17	2.5	3.7	3.41	3.13	3.81	3.46	3	3	6
+Goz-2017-600-491	491	şebek	macaque			3.42	2.63	3.36	4.06	3.58	4.14	5.56	1	2	5
+Goz-2017-600-492	492	şeffaf	transparent	3754	TRANSPARENT	3.59	2.37	3.6	4.4	3.94	4.35	4.37	12	2	6
+Goz-2017-600-493	493	şeker	sugar	862	SUGAR	1.41	1.38	1.27	1.51	1.45	6.18	6.63	143	2	5
+Goz-2017-600-494	494	şiddetli	severe	419	SEVERE	2.85	2.24	3.04	3.36	2.8	3.48	2.43	53	3	8
+Goz-2017-600-495	495	şok	shock			3.79	2.43	3.84	4.84	4.02	3.23	2.51	67	1	3
+Goz-2017-600-496	496	şüphe	suspicion	3197	SUSPICION	3.34	2.35	3.68	4	3.38	3.16	1.61	70	2	5
+Goz-2017-600-497	497	tabanca	gun	1566	GUN	2.07	1.84	2.02	2.36	2.04	6.44	6.93	72	3	7
+Goz-2017-600-498	498	tabu	taboo			5.11	2.92	5.55	5.66	5.8	3.2	2.25	14	2	4
+Goz-2017-600-499	499	tabure	stool	2058	STOOL	2.26	1.97	2.21	2.67	2.17	5.88	6.7	13	3	6
+Goz-2017-600-500	500	tahta	wood	1803	WOOD	1.95	1.81	2.04	2.26	1.72	6.24	6.94	155	2	5
+Goz-2017-600-501	501	takat	stamina			4.39	2.94	5.35	4.72	3.96	2.11	2.52	1	2	5
+Goz-2017-600-502	502	takoz	wedge			3.85	2.89	4.97	3.96	3.53	4.8	6.46	7	2	5
+Goz-2017-600-503	503	taksi	taxi			2.37	1.79	2.32	2.46	2.87	6.38	6.79	56	2	5
+Goz-2017-600-504	504	taksit	installment			4.13	2.52	4.01	4.85	5.02	3.92	4.58	89	2	6
+Goz-2017-600-505	505	tane	grain (of sand etc.)			1.85	1.75	1.84	2.03	1.79	3.76	4.76	273	2	4
```

Comment:
2086 GRAIN (CLASSIFIER) wouldn't fit here? Or is it only for a restricted type of languages and not meant for quantifiers?

Reply:
The "(CLASSIFIER)" concepts seem to be reserved for languages that have nominal classifiers (e.g., Chinese, Vietnamese).